### DISCUSSION TITLE
List of languages available on a user's profile

### DISCUSSION THREAD

**Discussion by alaurent-PS** [pimcore/pimcore#13014] [Q&A / Support] [answered]
Hello,

I have a problem on a Pimcore server.
I need some of my users to have the language "en_ZA" (English South Africa) set on their user profile.
However on one of my servers I don't have this language available in the user profile settings, while I do have it on other Pimcore machines (with the same version).

How can this language list be updated?
For information, the language "en_ZA" is added to the settings of Pimcore.

![image](https://user-images.githubusercontent.com/43468975/187635815-4f6da3ed-9f9a-4f5a-9035-580851798607.png)

Thanks for your help

**Comment by kingjia90** [comment, marked as answer]
Hello,

There is a clear separation between Frontend and Backend (Admin-interface) available and supported languages and they are not interchangeable.
![image](https://user-images.githubusercontent.com/6014195/187674905-1b8cffe4-e2a1-42b1-8322-ec649a930fbf.png)

In your case, you likely need to add a new own admin language, please see here `Adding your own admin languages` for more details:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Multi_Language_i18n/Admin_Translations.html#page_Adding-your-own-admin-languages-since-v6-3-6

Hope it helps

**Comment by alaurent-PS** [comment]
Hello @kingjia90 

The languages had already been added in the system parameters.
However your link allowed me to find where the translation files were stored (%kernel.project_dir%/translations/), I recovered the files from another machine and it works.

I still don't understand why these files weren't present, maybe an error with Git.

Anyway thank you for your help.
